src/lib/webpack-config.js

```diff
@@ -433,6 +435,7 @@ const production = config => addPlugins([
 		minify: true,
 		stripPrefix: config.cwd,
 		staticFileGlobsIgnorePatterns: [
+			/polyfills\.js$/,
```

Comment:
From what I understand `serviceWorker` implementation requires `fetch` implementation which in turn requires `Promise` implementation. Based on [FetchEvent](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent). I could be wrong tough.